crates/sc-consensus-subspace/src/archiver.rs

```diff
@@ -130,14 +131,14 @@ pub fn start_subspace_archiver<Block: BlockT, Client>(
             .runtime_api()
             .validated_object_call_hashes(&BlockId::Number(last_archived_block_number.into()))
             .expect("Block state must exist");
-        let block_object_mapping = client
-            .runtime_api()
-            .extract_block_object_mapping(
-                &BlockId::Number(last_archived_block_number.saturating_sub(1).into()),
-                last_archived_block.block.clone(),
-                validated_object_calls,
-            )
-            .expect("Must be able to make runtime call");
+
+        let block_object_mapping = ObjectsApi::extract_block_object_mapping(
+            client.runtime_api().deref(),
+            &BlockId::Number(last_archived_block_number.saturating_sub(1).into()),
+            last_archived_block.block.clone(),
+            validated_object_calls,
+        )
+        .expect("Must be able to make runtime call");
```

Comment:
We can't panic here though or else it will panic with old runtime. It should default to empty object mappings with a TODO to return `expect` in the future (or just print an error, no need to panic for these).